### PR TITLE
Updated default theme

### PR DESCRIPTION
**Pull request by zacdirect**
Minor UI updates to move away from the default Blazor theme and favicon.  Icon is kubernetes inspired and royalty free: https://en.wikipedia.org/wiki/File:Circle-icons-shipwheel.svg

### CODE REVIEW

**Comment by davidfowl** [comment]
Screenshot?

**Comment by davidfowl** [comment]
The about link looks a little clunky now:

image

**Comment by zacdirect** [comment]
Was going for a simple match on the nav.  Can keep it styled separately and have it be a bit more relaxed like so:
Chilled About

Thoughts?

**Comment by davidfowl** [comment]
Better

**Comment by zacdirect** [comment]
Checked in that change for your full review.  Thank you!

File: src/Micronetes.Hosting/wwwroot/css/site.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
 
 html, body {
-    font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-family: "Roboto",sans-serif;
```
Comment (PureKrome) [inline src/Micronetes.Hosting/wwwroot/css/site.css:4]: 🕺 💃 https://www.youtube.com/watch?v=uc6f_2nPSX8 

Mr. Roboto!

Comment (zacdirect) [reply src/Micronetes.Hosting/wwwroot/css/site.css:4]: > 🕺 💃 https://www.youtube.com/watch?v=uc6f_2nPSX8
> 
> Mr. Roboto!

In a world of open-source containers driven by automatic integration and deployment technologies, only one sans-serif font may reign supreme.

**Comment by davidfowl** [comment]
Thanks!

**Pull request merged**